tests/core/evm/vm/test_message.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import unicode_literals
+import pytest
+import evm.vm.message as m
+
+
+def _create_message(gas=1, gas_price=1, to=b"\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xff", sender=b"\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xff", value=1, data=b"", code=b"", origin="\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xff"):
```

Comment:
Can you pull all of these long bytestrings out into top level module constants.

```python
ADDRESS = b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xff'
```

Then you can just use them in the signature to make it readable.

```python
def _create_message(gas=1, to=ADDRESS, ...):
    ...
```